src/coreComponents/physicsSolvers/wavePropagation/WaveSolverUtils.hpp

```diff
@@ -308,10 +308,10 @@ struct WaveSolverUtils
                                         arrayView2d< wsCoordType const, nodes::REFERENCE_POSITION_USD > const nodeCoords,
                                         real64 (& coordsOnRefElem)[3] )
   {
-    real64 xLocal[FE_TYPE::numNodes][3]{};
-    for( localIndex a = 0; a < FE_TYPE::numNodes; ++a )
+    real64 xLocal[8][3]{};
```

Comment:
What was the problem with `FE_TYPE::numNodes`?

Reply:
This PR changes the way the jacobian matrix is computed for the GL hexahedra. It used to be computed based on all the nodes in the mesh element (including higher order nodes), which lead to a higher algorithmic complexity for higher orders. With this PR, only the actual mesh nodes are needed (which are always 8, independent of order). Thus, we don't need `FE_TYPE::numNodes` here anymore, since we are only using the 8 mesh vertices.
Note that since these are all SEM kernels, they can only work on hexahedra, so the number of nodes per face or element is fixed and will never be changed.